azure-pipelines/release.yml

```diff
@@ -51,6 +51,27 @@ extends:
           runOnce:
             deploy:
               steps:
+              - pwsh: |
```

Comment:
Downside is that this doesn't run until pretty late in the run but the resource variables aren't available until runtime and this is the earliest we can set it.